### DISCUSSION TITLE
Progress component

### DISCUSSION THREAD

**Discussion by clarkeash** [livewire/flux#636] [Feature Requests] [answered]
It would be great to have some kind of progress bar element, I know we have the html `<progress>` element but its rather limited
a `<flux:progress>` would be ideal, tooltip support etc would be nice.

**Comment by calebporzio** [comment]
Can you post more about what you imagine? maybe screenshots of other apps or libraries demonstrating what you want?

**Comment by clarkeash** [comment]
It seems like many examples dont have any form of tooltip, most seem to have no values or values at the end.
<img width="307" alt="Screenshot 2024-12-01 at 17 37 14" src="https://github.com/user-attachments/assets/ed90d70d-0849-4a6c-a42f-867987c9a283">
<img width="715" alt="Screenshot 2024-12-01 at 17 36 51" src="https://github.com/user-attachments/assets/68c20b67-51ff-4940-be72-1b861d1e1871">

This is nice but the value is there permanently (and the 100% spacing isnt great) ([link](https://preline.co/docs/progress.html))
<img width="754" alt="Screenshot 2024-12-01 at 17 42 27" src="https://github.com/user-attachments/assets/b11df30f-effa-4e6d-8b21-e371b5174c61">


I think this is the best example with the marker, the tooltip is display on hover of the marker ([link](https://tremor.so/docs/visualizations/category-bar#example))
<img width="440" alt="Screenshot 2024-12-01 at 17 29 43" src="https://github.com/user-attachments/assets/3c5ce569-9c24-4318-ad4b-f12e22a37f75">

**Comment by jdc1898** [comment]
Looks like this went in.

**Comment by joshhanley** [comment, marked as answer]
We've released time picker in v2.13.0!

See the blog post here https://fluxui.dev/blog/2026-02-27-progress-bar
And you can find the documentation here https://fluxui.dev/components/progress
